examples/auditLogProvider/src/interfaces/index.d.ts

```diff
@@ -14,7 +14,7 @@ export interface ILog {
     id: string;
     action: string;
     resource: string;
-    data: any;
-    previousData: any;
+    data: any; // eslint-disable-line
+    previousData: any; // eslint-disable-line
```

Comment:
```suggestion
    data: unknown;
    previousData: unknown;
```

I think `unknown` is a better fit than surpassing the warning, not sure if `unknown` is going to cause any issues but looks fine at first glance 😅

Reply:
done